scripts/serving/mlserver/mlserver/cli/build.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
Is it copied from seldon core?

Reply:
Yep, the first commit add the source code from mlserver

Reply:
/cc @simon-cj